components/organisms/ContributorProfileTab/contributor-profile-tab.tsx

```diff
@@ -17,9 +17,18 @@ import Button from "components/atoms/Button/button";
 import useSupabaseAuth from "lib/hooks/useSupabaseAuth";
 import uppercaseFirst from "lib/utils/uppercase-first";
 import useFetchAllEmojis from "lib/hooks/useFetchAllEmojis";
+import { getInterestOptions, interestsType } from "lib/utils/getInterestOptions";
+import { updateUser } from "lib/hooks/update-user";
+import { useToast } from "lib/hooks/useToast";
 
 import PaginationResults from "components/molecules/PaginationResults/pagination-result";
 import Pagination from "components/molecules/Pagination/pagination";
+import DashContainer from "components/atoms/DashedContainer/DashContainer";
+import LanguagePill from "components/atoms/LanguagePill/LanguagePill";
+import RecommendedRepoCard from "components/molecules/RecommendedRepoCard/recommended-repo-card";
+import recommendations from "lib/utils/recommendations";
+import Title from "components/atoms/Typography/title";
+import clsx from "clsx";
```

Comment:
Move to top with third-party imports
```suggestion
import clsx from "clsx";
```